MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -25,6 +25,14 @@ class RouteManeuverViewController: UIViewController {
         }
     }
     
+    var leg: RouteLeg? {
+        didSet {
+            if let _ = roadCode {
```

Comment:
If `leg` is set but there’s no `roadCode`, the street name doesn’t get updated. I’m not sure I understand what `roadCode` has to do with the leg.